src/warden/src/bin/reftest.rs

```diff
@@ -185,5 +225,6 @@ fn main() {
         num_failures += harness.run(instance);
     }
     let _ = harness;
+    num_failures += 0;
```

Comment:
`+= 0`?

Reply:
This suppresses rust warning for unused `mut`